library/ssl_tls.c

```diff
@@ -4751,10 +4751,16 @@ int mbedtls_ssl_conf_dtls_srtp_protection_profiles( mbedtls_ssl_config *conf,
     return( 0 );
 }
 
-const mbedtls_dtls_srtp_info *
+mbedtls_dtls_srtp_info
      mbedtls_ssl_get_dtls_srtp_negotiation_result( const mbedtls_ssl_context *ssl )
 {
-    return( &( ssl->dtls_srtp_info ) );
+    mbedtls_dtls_srtp_info ret = ssl->dtls_srtp_info;
```

Comment:
Nit: `ret` is always used for numeric error codes from what I remember. I'd prefer a different name. Certainly not a blocker.

Reply:
This is outdated code, last version does not use ret value as it gets an output argument